sdk/nodejs/runtime/closure/v8.ts

```diff
@@ -39,15 +40,23 @@ interface V8Script {
     locationFromPosition(pos: V8SourcePosition): V8SourceLocation;
 }
 
+// Check if we're on node 10.  We need to call specific intrinsics depending on which version we're on.
+const isNodeV10 = semver.gte(process.version, "10.0.0");
```

Comment:
NIT: This should actually be named isNodeV10OrLater?

Reply:
Sorry!  Didn't see this in the conversation list.  Your name is def better.